depends/Makefile

```diff
@@ -154,6 +154,14 @@ check-packages: check-sources
 install: check-packages $(host_prefix)/share/config.site
 
 
+download-crates: native_rust
+	$(AT)rm -rf $(host_prefix)
```

Comment:
This [makes me slightly nervous](https://github.com/valvesoftware/steam-for-linux/issues/3671) (and the other one with `$(@D)` above, although I know that's preexisting). Where/how are `$(host_prefix)` and `$(@D)` set?

Reply:
In the first one, `$(@D)` is the directory in which this Makefile step's file is located; ergo for step `$(host_prefix)/.stamp_$(final_build_id):` it is `$(host_prefix)`.

`$(host_prefix)` is set [here](https://github.com/zcash/zcash/blob/6f9b281ef7a5bc28a99d56e97106489b6314fd6b/depends/Makefile#L61-L63), and thus is guaranteed to begin with `$(BASEDIR)`.